tests/acl/test_acl.py

```diff
@@ -437,7 +437,7 @@ def setup(duthosts, ptfhost, rand_selected_dut, rand_selected_front_end_dut, ran
             )
             or 't1-isolated' in tbinfo["topo"]["name"]
         )
-        and not re.match(r"t0-.*-s\d+$", tbinfo["topo"]["name"])
+        and not re.match(r"t0-.*-s\d+", tbinfo["topo"]["name"])
```

Comment:
hi Ziting, I believe this is the actual issue:

![image](https://github.com/user-attachments/assets/82f08718-7509-468e-8c25-ba4ea81ce56d)